pkg/v1/tarball/write.go

```diff
@@ -191,6 +193,10 @@ func processImages(refToImage map[name.Reference]v1.Image, tf *tar.Writer) (Mani
 			LayerSources: layerSources,
 		})
 	}
+	// sort by name of the repotags so it is consistent. Alternatively, we could sort by hash of the
```

Comment:
It's a small thing, but I find `sort.Slice` simpler and easier to read -- it doesn't require satisfying an interface, it uses reflection instead. It shouldn't really affect performance while there are only going to be a handful of items.

Reply:
Sure, I don't mind. Hang tight.

Reply:
Done